tests/checker/ftpserver.py

```diff
@@ -50,31 +50,34 @@ def tearDown (self):
 
 
 def start_server (host, port):
-    def line_logger(msg):
+    def line_logger(self, msg):
         if "kill" in msg:
             raise KeyboardInterrupt()
 
     try:
-        from pyftpdlib import ftpserver
+        from pyftpdlib.authorizers import DummyAuthorizer
+        from pyftpdlib.handlers import FTPHandler
+        from pyftpdlib.servers import FTPServer
+        from pyftpdlib import __ver__ as pyftpdlib_version
     except ImportError:
         pytest.skip("pyftpdlib is not available")
         return
-    authorizer = ftpserver.DummyAuthorizer()
+    authorizer = DummyAuthorizer()
     datadir = os.path.join(os.path.dirname(__file__), 'data')
     authorizer.add_anonymous(datadir)
 
     # Instantiate FTP handler class
-    ftp_handler = ftpserver.FTPHandler
+    ftp_handler = FTPHandler
```

Comment:
Oh my, the comment is lying to us!  There's no class instantiation here, just some monkeypatching.  Interesting.

(Not relevant to your PR, unless we maybe want to take this opportunity and fix the misleading comment?)

Reply:
can be punted